package.json

```diff
@@ -115,7 +115,7 @@
             "./README.md",
             "./docs/api.md",
             "./demo/demo.md",
```

Comment:
Update this to `index.md` and then this is approved

Reply:
@jordanjones243 you are going to have to help @braven112 with the change request you want. You instructed him to change the ./demo/demo.md file name, which he did, but that is wrong change. This should have been a change to the _partials/demo.md as well as the generateDocs.js script to build index.md instead of demo.md.

All in all, that change may be more than we should expect Brandon to do. Either help him make that change directly or let's back it out of this PR and do it on our own as a tech debt ticket.